src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -93,6 +93,8 @@ public enum KeyBinding {
     PULL_CHANGES_FROM_SHARED_DATABASE("Pull changes from shared database", Localization.lang("Pull changes from shared database"), "ctrl+shift+R", KeyBindingCategory.FILE),
     PREVIOUS_PREVIEW_LAYOUT("Previous preview layout", Localization.lang("Previous preview layout"), "shift+F9", KeyBindingCategory.VIEW),
     PREVIOUS_LIBRARY("Previous library", Localization.lang("Previous library"), "ctrl+PAGE_UP", KeyBindingCategory.VIEW),
+    SCROLL_TO_NEXT_RANK("Scroll to next rank", Localization.lang("Scroll to next rank"), "right", KeyBindingCategory.VIEW),
```

Comment:
JabRef uses the term "rank" for star ranking:

![image](https://github.com/user-attachments/assets/3082c181-2adc-4c57-8c5c-982dd9f5c822)

https://docs.jabref.org/finding-sorting-and-cleaning-entries/specialfields#ranking

Also refs https://github.com/JabRef/jabref-issue-melting-pot/issues/72

Please rename.

Reply:
Rename to `MatchCategory`. 

Based on a conversation with ChatGPT: https://chatgpt.com/share/f9163e63-d77f-4ff3-bd16-73091cad8495

Reply:
Should I rename this method only to `scrollToNextSearchRank`, or change all occurrences of "rank" to something like "matchState"?

Reply:
Done in: [`ce71705` (#11510)](https://github.com/JabRef/jabref/pull/11510/commits/ce71705bc597d86a95a11aefe167416411243bee)